android/java/res/layout/fragment_portfolio.xml

```diff
@@ -21,15 +21,14 @@
                 android:orientation="horizontal">
 
                 <TextView
-                    android:id="@+id/balance"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
                     android:layout_weight="1"
                     android:paddingStart="16dp"
                     android:paddingEnd="16dp"
                     android:paddingTop="16dp"
                     android:layout_marginBottom="8dp"
-                    android:text="@string/brave_wallet_portfolio_balance_updating"
+                    android:text="$12,453.17"
```

Comment:
What is this number?

Reply:
This is a hard-coded number from design specs.
It re-appeared here now because I had to revert whole PR (https://github.com/brave/brave-core/commit/31653d9faff601e70a755a29a187bc1793cce72b)  which was using `CompletableFuture` .